test/linter/test_yaml_actionlint.vader

```diff
@@ -28,6 +27,21 @@ Execute(Problems should be parsed correctly for actionlint):
   \   'workflow_call_event.yaml:56:23: property "unknown_input" is not defined in object type {input7: bool; input0: any; input1: any; input2: string; input3: any; input4: any; input5: number; input6: number} [expression]',
   \ ])
 
+Execute(Shellcheck issues should be reported at the line they appear):
```

Comment:
I added a new test instead of adding a new entry to the existing test, to make it easier to spot if it's the default implementation, or the shellcheck-specific one that fails.